prboom2/src/g_game.c

```diff
@@ -2004,6 +2004,35 @@ unsigned int GetPackageVersion(void)
   return PACKAGEVERSION;
 }
 
+// [FG] support named complevels on the command line, e.g. "-complevel boom",
+// names based on the args to Chocolate Doom's "-gameversion" parameter
+
+static int GetComplevel (const char *arg)
+{
+  int i;
+
+  struct {
+    const int level;
+    const char *const name;
+  } named_complevel[] = {
+    {2, "1.9"},
+    {3, "ultimate"},
+    {4, "final"},
```

Comment:
We could have extra aliases here as well, like "tnt" and "plutonia". Another would be "doom" for cl 3. That way you can basically pass the iwad name for the complevel.

Reply:
Very good idea! I'll add the IWAD names as aliases. I am a bit unsure about plain "doom", though, as that's a bit too ambiguous for my taste. The original non-Ultimate Doom had the "doom2" executable after all.

Reply:
It could be ambiguous, I'm not sure. I'd probably still use it but i don't have a super strong opinion on it.

Reply:
If you want to be even more aggressive on this feature we could just have vanilla, boom, and mbf, where vanilla chooses the complevel based on the iwad.

Reply:
> If we want to be even more aggressive on this feature we could just have vanilla, boom, and mbf, where vanilla chooses the complevel based on the iwad.

But isn't that the standard behavior anyway?

Reply:
> But isn't that the standard behavior anyway?

For demos, yes, but not for regular game play.